.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main   
+  pull_request:  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          components: clippy
+
+      - name: Install cargo-hack
+        run: cargo install cargo-hack
```

We can use `taiki-e/install-action` for pulling binaries instead.

```suggestion
      - name: Install cargo-hack
        uses: taiki-e/install-action@v2
        with:
          tool: cargo-hack
          checksum: true

```